### PR TITLE
Solution patterns RHTE small fixes

### PR DESCRIPTION
**Pull request by mayorova**

### CODE REVIEW

File: walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc [hunk 1]
```diff
@@ -435,6 +435,7 @@ After setting up the integration between the Order Entry System and Order Manage
 :spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
 :fuse-url: https://eval.apps.city.openshiftworkshop.com/
 
+// TODO: currently the app has the login screen, either remove the screen in the app and allow free access, or instruct the user here to enter a username ([A-Za-z]{1,15}) and click *Log in*.
```
Comment (evanshortiss) [inline walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc]: Yeah, I meant to remove that. Will remove it before the lab session, thanks for the reminder!

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc [hunk 2]
```diff
@@ -1,11 +1,11 @@
-:sso-realm-url: https://sso-user-sso.{{openshift-app-host}/auth/admin/{user-username}/console/index.html
+:sso-realm-url: {user-sso-url}/auth/admin/{user-sanitized-username}/console/index.html
```
Comment (evanshortiss) [inline walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc:1]: Oh is this a new variable I missed? 👍

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc [hunk 3]
```diff
@@ -160,6 +160,7 @@ this may not be advisable._
 . Select *Overview* on the left and find the *rhmi-lab-nodejs-order-frontend* in the list.
 . Wait for it to finish deploying.
 . Click the route listed beside the *rhmi-lab-nodejs-order-frontend* to open the *Order Entry System*. A login screen with the title *{user-username} Realm* is displayed.
+// TODO: Verify this... In my case, the user `evals03` was already logged in (the name in top-right corner), I assume because there was a session in RH-SSO. Or maybe instruct here to open in the private browser session (like in the verification step).
```
Comment (evanshortiss) [inline walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc]: That's interesting, but might be linked to the SSO session as you said. I think we recommend the private session by default to "avoid potential session issues"?

**Review by evanshortiss — COMMENTED** [review]
Thanks @mayorova, really nice fixes. Just one or two comments from me.

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc (hunk 2 above)
Comment (mayorova) [reply walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc:1]: It's right here ;) 
https://github.com/evanshortiss/integreatly-vscode-asciidoc-snippets/blob/eb9f91c/.vscode/integreatly.code-snippets#L87

File: walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc (hunk 3 above)
Comment (mayorova) [reply walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc]: Sure.
Change the `Click the route... ` item to:

```
Open the URL listed beside the *rhmi-lab-nodejs-order-frontend* in  a private browser session to open the *Order Entry System*. A login screen with the title *{user-username} Realm* is displayed.
``` 
?

**Pull request merged**